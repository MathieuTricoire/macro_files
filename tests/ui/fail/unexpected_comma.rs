use macro_files::create;

fn main() {
    create!({
        "README.md",
        ".gitignore": "/target"
    });
}
