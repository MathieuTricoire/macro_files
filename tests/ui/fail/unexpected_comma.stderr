error: no rules expected the token `,`
 --> tests/ui/fail/unexpected_comma.rs:5:20
  |
5 |         "README.md",
  |                    ^ no rules expected this token in macro call
